Assets/MRTK/Core/Services/MixedRealityToolkit.cs

```diff
@@ -88,14 +93,38 @@ public MixedRealityToolkitConfigurationProfile ActiveProfile
             }
             set
             {
-                ResetConfiguration(value);
+                if(Application.isPlaying && activeProfile != null && value != null && activeProfile != value)
+                {
+                    newProfile = value;
+                }
+                else
+                {
+                    ResetConfiguration(value);
```

Comment:
If I happened to pass the same profile in that matches the current one, we'd still want to reset in that case right?